### PR TITLE
add demo/ folder with examples

### PR DESCRIPTION
**Pull request by aantn**

### CODE REVIEW

**Comment by aantn** [comment]
@arikalon1 note that I didn't document all the demos yet. There is still some work to do here.

File: demo/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+1. You can get Slack notifications about issues in your cluster 
+2. Notifications contain the exact data you need to debug
+3. You see more troubleshooting data in the Robusta UI, including recent changes to Kubernetes objects
+
+# OOM Kill Demo
+
+## Running it
+
+1. Run `kubectl apply -f ./oomkill_demo/pod_oomkill.yaml`
+2. Check your Slack channel for a notification about the crashing pod. The message includes details you need to debug the OOM
```
Comment (arikalon1) [inline demo/README.md:26]: Is this the crashing pod or oom killed pod?

File: demo/custom_playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (arikalon1) [inline demo/custom_playbook.yaml:1]: Is this file used anywhere here?

**Review by arikalon1 — APPROVED** [review]
Nice work
Left 2 minor comments

File: demo/README.md (hunk 1 above)
Comment (aantn) [reply demo/README.md:26]: What do you mean?

File: demo/custom_playbook.yaml (hunk 2 above)
Comment (aantn) [reply demo/custom_playbook.yaml]: No, removed it.

File: demo/README.md (hunk 1 above)
Comment (arikalon1) [reply demo/README.md:26]: It's oom kill demo, and the notification should be oom kill enricher, right?
The text says "for a notification about the crashing pod"
Should this be "for a notification about the oom killed pod" ?

**Pull request merged**